packages/terra-avatar/src/terra-dev-site/doc/avatar/About.1.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+<Badge />
+
+# About terra-avatar
```

Comment:
It looks like we copied over the content from `docs/README.md/`. Do we need to delete `'../../../../docs/README.md';`?

Reply:
Oops I didn't see it. I will change it right now.

Reply:
updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6